backend/windmill-worker/src/python_executor.rs

```diff
@@ -1322,7 +1322,11 @@ pub async fn handle_python_reqs(
         } else {
             let fssafe_req = NON_ALPHANUM_CHAR.replace_all(&req, "_").to_string();
             #[cfg(unix)]
-            let req = format!("'{}'", req);
```

Comment:
sure about that change?

Reply:
Yes, if we dont do this uv will treat ' as an actual package name for some reason. ' also ignored on windows workers, since they already call uv directly.

Reply:
That what happens if we don't do this:
```
absl-py==2.1.0 is being installed for the first time.
 It will be cached for all ulterior uses.
error: Failed to parse: `'absl-py==2.1.0'`
  Caused by: Expected package name starting with an alphanumeric character, found `'`
'absl-py==2.1.0'
^
```